bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/transport/HubIOStream.java

```diff
@@ -47,11 +51,13 @@ public class HubIOStream extends IOStream {
 
     private static final String BS_START = "<BS>";
     private static final String BS_END = "</BS>";
+    private static final int REQUEST_TIMEOUT = 30; // in seconds
```

Comment:
Consider:
```suggestion
    private static final int REQUEST_TIMEOUT_SECONDS = 30;
```

This will make it harder to miss where used.

Reply:
There is a `TimeUnit` specified where it is used. The comment is more for context and consistent with how I defined similar constant in the binding.

https://github.com/openhab/openhab-addons/blob/e3ac1ad2b6709dbac6f1b738c6d81c2a52a46b08/bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/transport/HubIOStream.java#L281-L282

Reply:
OK.